packages/0xcert-ethereum-proxy-contracts/src/tests/nftokens-safe.test.ts

```diff
@@ -83,6 +84,35 @@ spec.test('transfers an NFT', async (ctx) => {
   ctx.is(newOwner, sara);
 });
 
+spec.test('transfers an NFT high ID', async (ctx) => {
```

Comment:
"transfers a high ID NFT" or "transfers an NFT with high ID" would be more accurate.
This applies to all descriptions.